packages/edit-post/src/components/fullscreen-mode/index.js

```diff
@@ -7,6 +7,12 @@ import { withSelect } from '@wordpress/data';
 class FullscreenMode extends Component {
 	componentDidMount() {
 		this.sync();
+
+		// `is-fullscreen-mode` is set in PHP as a body class by Gutenberg, and this causes
+		// `sticky-menu` to be applied by WordPress and prevents the admin menu being scrolled
+		// even if `is-fullscreen-mode` is then removed. Let's remove `sticky-menu` here as
+		// a consequence of the FullscreenMode setup
+		document.body.classList.remove( 'sticky-menu' );
```

Comment:
Shouldn't this be added back once the component is unmounted? Otherwise loading full screen mode will remove the sticky-menu class forever (well, for the rest of this browser tab's session).

Sidenote: this kind of direct DOM manipulation in components scares me and I think warrants a test, but I get why it's being done and it makes sense.